examples/sphere/shallow_water.jl

```diff
@@ -477,15 +489,9 @@ function shallow_water_driver(ARGS, usempi::Bool, ::Type{FT}) where {FT}
     T = 86400 * 2
 
     prob = ODEProblem(rhs!, Y, (0.0, T), parameters)
-    integrator = OrdinaryDiffEq.init(
-        prob,
-        SSPRK33(),
-        dt = dt,
-        saveat = dt,
-        progress = true,
-        adaptive = false,
-        progress_message = (dt, u, p, t) -> t,
-    )
+    integrator_args = (prob, SSPRK33(), dt, dt, true, false)
+    integrator = get_integrator(integrator_args...)
```

Comment:
`integrator` needs to be defined at global scope for the allocations script to work, right now it's only inside the scope of `shallow_water_driver`

Reply:
Ah, nvm, I missed what was done below. But the
```julia
    if haskey(ENV, "CI_PERF_SKIP_RUN") # for performance analysis
        throw(:exit_profile)
    end
```
has to be in global scope right after the integrator is defined in global scope, so we need to delete the `if .. CI_PERF_SKIP_RUN ..` a few lines down from here.